### DISCUSSION TITLE
History of Downloads in the Monitoring  section

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#374] [Ideas]
Hi,

Just a thought from my side. Most of the time there is nothing to be seen in the MONITORING section. Only if a DOWNLOAD is in progress.

I wonder if it is an idea to also display the downloads finished in that section. Some kind of history of DOWNLOADS ordered from the most recent one at the top. If there is one of more downloads in progress just diplay them at the top of the history.

Kind regards,
Henk - ftp.ph5hp.nl

**Comment by rejetto** [comment]
hey, you are supposed to use the "logs" page for the history of downloads.
Is anything wrong with that?

**Reply by PH5HP** [reply]
Hi,

No, nothing wrong with that. It was just a suggestion. I know that it can be seen in log page, but I thought because most of the time  not much can be seen in the monitor page other then a ongoing download maybe it was an idea to put the download history also there.

As I said before it is just a thought. :-) 

Kind regards,
Henk - ftp.ph5hp.nl
